CMakeLists.txt

```diff
@@ -180,10 +184,16 @@ elseif (CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_COMPILER_ID STREQUAL "Clang")
 		SET(CMAKE_EXE_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} -fprofile-arcs -ftest-coverage")
 		SET(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS}  -fprofile-arcs -ftest-coverage")
 	endif()
-	if(DEFINED ENV{CI} AND (DEFINED ENV{TRAVIS} OR DEFINED ENV{APPVEYOR}))
-		message(STATUS "Running in CI environment (Travis or AppVeyor), setting -Werror")
+	if(DEFINED ENV{TF_BUILD} OR (DEFINED ENV{CI} AND (DEFINED ENV{TRAVIS} OR DEFINED ENV{APPVEYOR})))
+		message(STATUS "Running in CI environment (Azure, Travis or AppVeyor), setting -Werror")
 		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror")
+		if (CMAKE_C_COMPILER_ID STREQUAL "Clang" AND CMAKE_C_COMPILER_VERSION VERSION_EQUAL "11.0")
+			set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-missing-braces")
```

Comment:
This is new, and not talked about in the commit message, what's that for?

Reply:
yeah, this was for older versions of clang. 

In gcc - this warning was turned off in 4.8.0 (2005)

I will add something in the commit message, or better yet - split it up into a separate commit.

Reply:
If we only enable -Werror for CI, and this a workaround for an old version of Clang, do we still need it?

Reply:
looks like it was only needed for the version of MacOS, that we removed...

So - it's gone now.